src/index.ts

```diff
@@ -84,7 +86,8 @@ export default function supernova(env: Galaxy) {
       });
 
       usePaginationTable({
-        env,
+        isNewHeadCellMenuEnabled,
+        env, // TODO: this is not used!
```

Comment:
should we remove this?

Reply:
did you remove the usage of it in this PR?

Reply:
no, i was just asking if we can remove it because seems like it hasn't been used

Reply:
then that is a different PR I would say